src/overlays/actors/ovl_En_Heishi/z_en_heishi.c

```diff
@@ -15,9 +16,21 @@ void EnHeishi_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void EnHeishi_Update(Actor* thisx, GlobalContext* globalCtx);
 void EnHeishi_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-void func_80BE9214(EnHeishi* this, GlobalContext* globalCtx);
+void EnHeishi_ChangeAnimation(EnHeishi* this, s32 animIndex);
+void EnHeishi_SetHeadRotation(EnHeishi* this);
+void EnHeishi_SetupIdle(EnHeishi* this);
+void EnHeishi_Idle(EnHeishi* this, GlobalContext* globalCtx);
+s32 EnHeishi_OverrideLimbDraw(GlobalContext* globalctx, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot,
+                              Actor* thisx);
```

Comment:
```suggestion
```
No need to forward declare this function, as it is only used after it has been defined.